buildpack/telemetry/telegraf.py

```diff
@@ -255,10 +258,20 @@ def update_config(m2ee, app_name):
     template_path = os.path.join(_get_config_file_dir(version), TEMPLATE_FILENAME)
 
     tags = util.get_tags()
+
     if datadog.is_enabled() and "service" not in tags:
         # app and / or service tag not set
         tags["service"] = datadog.get_service_tag()
 
+    # Add application tags to the custom metrics sent to New Relic
+    if newrelic.is_enabled():
+        newrelic_tags = newrelic.get_metrics_tags(app_name)
+        print(f"NR_TAGS: {newrelic_tags}")
+        # Makes sure the user defined values persist, if the tags overlap.
+        newrelic_tags.update(tags)
+        tags = newrelic_tags
+        print(f"TAGS: {tags}")
+
```

Comment:
forgot to remove `print` ?

Reply:
Think you got an in-between commit in which I was testing that, but it's gone now ;).